config/filesystems.php

```diff
@@ -45,9 +45,7 @@
 
         'local' => [
             'driver' => 'local',
-            'root' => storage_path('app'),
-            //            'root' => storage_path('app'),
-
+            'root' => public_path(),
```

Comment:
Why are you change this?

Reply:
I chacnged this one because Yaroslav Plaxiy showed this way of storing images into 'upload' directory.  I don`t know other ways, sorry